integration/common.go

```diff
@@ -17,33 +17,39 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-type fileUploadSuite struct {
+// FileUploadSuite tests the file upload functionalities, using a provided configuration
+type FileUploadSuite struct {
 	suite.Suite
 	util.SuiteInitializer
 
-	thingURL   string
-	featureURL string
-	uploadCfg  uploadTestConfig
+	ThingURL   string
+	FeatureURL string
+
+	uploadCfg UploadTestConfig
```

Comment:
As an example, we have _BackupAndRestoreTestConfig_, which embeds _UploadTestConfig_(its almost the same, but with _backupAndRestoreCfg_ instead of _uploadCfg_, because of a different environment variables naming).